doc/source/whatsnew/v0.24.0.txt

```diff
@@ -1296,6 +1296,7 @@ Groupby/Resample/Rolling
 - :func:`RollingGroupby.agg` and :func:`ExpandingGroupby.agg` now support multiple aggregation functions as parameters (:issue:`15072`)
 - Bug in :meth:`DataFrame.resample` and :meth:`Series.resample` when resampling by a weekly offset (``'W'``) across a DST transition (:issue:`9119`, :issue:`21459`)
 - Bug in :meth:`DataFrame.expanding` in which the ``axis`` argument was not being respected during aggregations (:issue:`23372`)
+- Bug in :meth:`DataFrame.groupby.transform` which caused missing values when the input function can accept a :class:`pandas.DataFrame` but renames it (:issue:`23461`).
```

Comment:
I think this needs to be
``:func:`pandas.core.groupby.DataFrameGroupBy.transform` `` to render correctly

Reply:
:+1: